### DISCUSSION TITLE
Ideas for minigames

### DISCUSSION THREAD

**Discussion by DarkGlamFinch** [wd357dui/Elements-of-Harmony#11] [Ideas]
Set Higher difficulty options
Increase replayability
Make more random, addictive, and infinite/endless the actual minigames for single player
Play with any pony and clone them in new colors.

Bigger ideas:
Like a Bubble shooter clone campaign
Like a Defense Tower clone mode campaign
Like a Vampire Survivor clone mode campaign
Like Wario  Ware minigames or Mario Party minigames in repeteable campaign

**Comment by wd357dui** [comment]
It's great to see suggestions for the mod, my idea of workflow on adding new minigames was that new minigames should be made in an independently created Unity project, and later be ported to the game mod, which requires some "reverse engineering" to know how Unity works; that's not the hard part, with my success rate in the past I'm positive this can be done eventually (although it may be time-consuming). **The hard part is, that I've never actually developed games before,** I see those suggestions, but I can't think of an idea on how to implement them exactly, I feel that when I get to it in the future I could (maybe) ask someone I know, like the talented game dev [WonderLS](https://www.youtube.com/channel/UCJ3CHsirD9HURrc2NRhITPg), or watch youtube tutorials or something, but overall, actual game development is more difficult than the "reverse engineering" part for me.

The bad news is, due to my college's new semester starting, my very low R&D efficiency, my other projects, and probably also due to the mod's lack of attraction in the fandom, I lost the motivation to devote any more of my (already diminishing amount of) free time to continue this project (again)... It might be better if more programmers would join the development, but right now, this project is... practically on hold.
